modules/cnv_connecting_resource_bridge_network.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_attaching_resource_multiple_networks.adoc
+
+[id="connecting-resource-bridge-network-{context}"]
+= Connecting a resource to a bridge network
+
+As a network administrator, you can configure a NetworkAttachmentDefinition
+of type `cnv-bridge` to provide Layer-2 networking to Pods and virtual machines.
+
+.Prerequisites
+
+* {ProductName} 2.0 or newer
+* A Linux bridge must be configured on every host.
+* `vlan_filtering` must be enabled on the bridge.
+
+.Procedure
+
+. Create a new file for the NetworkAttachmentDefinition. This file can be stored
+locally in any directory.
++
+----
+$ touch network-attachment-definition.yaml
```

Comment:
the `touch` stage seems superfluous to me. Can we just merge it into the Edit bullet point, E.g

> Create a new file for the NetworkAttachmentDefinition with your favorite. This file can be stored
locally in any directory.